examples/intro_data_profiler.ipynb

```diff
@@ -0,0 +1,442 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "60af6856",
+   "metadata": {},
+   "source": [
+    "# DataProfiler - What's in your data?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3297611c",
+   "metadata": {},
+   "source": [
+    "This introductory jupyter notebook demonstrates the basic usages of the DataProfiler. The library is designed in the way that users can easily get the statistics and other detailed information about the input datasets with just several lines of code. DataProfiler provides multiple data classes that can handle different types of data. In addition, users are given various options to skip some properties if not needed while profiling their datasets. The last key feature covered in this example is the ability to allow users to update their profilers from multiple batches of the large datasets, or merge multiple profilers which is suitable for the distributed computing environment. In particular, this example covers the followings:\n",
+    "\n",
+    "    - Basic usage of DataProfiler\n",
+    "    - Profiler options\n",
+    "    - Data reader class\n",
+    "    - Update profiles and merge profiles\n",
+    "\n",
+    "First, let's import the libraries needed for this example."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d4b774f6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "import json\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "sys.path.insert(0, '..')\n",
+    "import dataprofiler as dp\n",
+    "\n",
+    "data_path = \"../dataprofiler/tests/data\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4054a3df",
+   "metadata": {},
+   "source": [
+    "## Basic examples"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fc7083d8",
+   "metadata": {},
+   "source": [
+    "This section shows the basic example of DataProfiler. A CSV dataset is read using the data reader, then the returned data is given to the DataProfiler to obtain the properties and stastistics."
+   ]
```

Comment:
I think the header should be more detailed than "Basic examples".
I think you should say Data object instead of returned data.
I think properties is vague
"statistics" typo